cardano-node/src/Cardano/Tracing/Config.hs

```diff
@@ -24,13 +23,13 @@ import           Data.Aeson.Types (Parser)
 import           Data.Text (pack)
 
 import           Cardano.BM.Tracing (TracingVerbosity (..))
-
 import           Cardano.Node.Orphans ()
 
 
 data TraceOptions
   = TracingOff
   | TracingOn TraceSelection
+  | TraceDispatcher TraceSelection
```

Comment:
Let's change `TracingOn` to `TracingOnLegacy`. Do we have a deprecation timeline?

Reply:
Done

Reply:
I don't know a specific timeline.

Reply:
@Jimbo4350, this is a community management question, first and foremost.

Reply:
We'll be deciding the timeline for retirement of the old system after the public release.